packages/web/src/components/page/Page.tsx

```diff
@@ -10,8 +10,7 @@ import {
 
 import { Nullable } from '@audius/common/utils'
 import cn from 'classnames'
-// eslint-disable-next-line no-restricted-imports -- TODO: migrate to @react-spring/web
-import { Spring } from 'react-spring/renderprops.cjs'
+import { animated, useSpring } from 'react-spring'
```

Comment:
extra credit if we can use @react-spring/web?

Reply:
done!